### PR TITLE
Prepare for CICD

### PR DESCRIPTION
**Pull request by werbth**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/main/java/org/entando/plugins/pda/core/engine/Connection.java [hunk 1]
```diff
@@ -3,29 +3,27 @@
...
 import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
-import lombok.ToString;
 
-@Getter
```
Comment (ffleandro) [inline src/main/java/org/entando/plugins/pda/core/engine/Connection.java:12]: Connection não deveria ter `@Data` pois não serializado/deserializado.
`@Getter @Setter` é mais correto.

**Review by ffleandro — CHANGES_REQUESTED** [review]

File: src/main/java/org/entando/plugins/pda/core/engine/Connection.java (hunk 1 above)
Comment (werbth) [reply src/main/java/org/entando/plugins/pda/core/engine/Connection.java:12]: `@Data` has nothing to do with serialization/deserialization requirements. According to its Javadoc:
```
Generates getters for all fields, a useful toString method, and hashCode and equals implementations that check all non-transient fields. Will also generate setters for all non-final fields, as well as a constructor.
Equivalent to {@code @Getter @Setter @RequiredArgsConstructor @ToString @EqualsAndHashCode}.
```
So, every time you have all those annotations together, you can and you should replace for `@Data`. More details here: https://projectlombok.org/features/Data

Comment (ffleandro) [reply src/main/java/org/entando/plugins/pda/core/engine/Connection.java:12]: Cool, nice to know this!

**Review by ffleandro — APPROVED** [review]

**Pull request merged**